plugins/module_utils/kibana.py

```diff
@@ -96,7 +104,7 @@ def send_kibana_api_request(self, endpoint, method, data=None, headers={}, timeo
       else:
         return response_list[0]
   
-  def send_epr_api_request(self, endpoint, method, data=None, headers={}, timeout=120):
+  def send_epr_api_request(self, endpoint, method, data=None, headers={}, timeout=600):
     url = f'https://epr.elastic.co/{endpoint}'
```

Comment:
So this isn't important right now, but at some point we may want to change the URL here to an input with a default, custom package repos are technically a thing that's supported